docs/proposals/diagnose-experience.md

```diff
@@ -29,12 +29,12 @@ At first, the output of ORAS flag `--verbose` and `--debug` should be clarified
 
 There are three types of output in ORAS CLI:
 
-- **Status output**: such as progress information, progress bar in pulling or pushing files 
-- **Metadata output**: showing what has been pulled (e.g. filename, digest, etc.) in specified format, such as JSON, text
-- **Content output**: it is to output the raw data obtained from the remote registry server or file system, such as the pulled artifact content save as a file
+- **Status output**: such as progress information, progress bar in pulling or pushing files.
+- **Metadata output**: showing what has been pulled (e.g. filename, digest, etc.) in specified format, such as JSON, text.
+- **Content output**: it is to output the raw data obtained from the remote registry server or file system, such as the pulled artifact content save as a file.
+- **Error output**: error message are expected to be helpful to troubleshoot where the user has done something wrong and the program is guiding them in the right direction.
 
-
-The target users of these types of output are normal users. Currently, the output of ORAS `--verbose` flag belongs to status output since it contains detailed progress status. The output of `--verbose` only exists in oras `pull/push/attach` commands. 
+The target users of these types of output are standard users. Currently, the output of ORAS `--verbose` flag only exists in oras `pull/push/attach/discover` commands, which prints out detailed status output and metadata output. 
```

Comment:
nit: Instead of "standard users", how about "regular users"?

Reply:
@Wwwsylvia  "General users" or "Regular Users" looks good to me since these terms are commonly used in Linux.  Or we can also consider using "user" and "developer" to differentiate two types of personas.

Reply:
`general users` sounds better.